axelor-account/src/main/java/com/axelor/apps/account/web/AccountChartController.java

```diff
@@ -68,4 +74,19 @@ public void installChart(ActionRequest request, ActionResponse response) throws
 
     } else response.setFlash(I18n.get(IExceptionMessage.ACCOUNT_CHART_3));
   }
+
+  public void setAdminExpAccountType(ActionRequest request, ActionResponse response) {
+
+    AccountConfig accountConfig = request.getContext().asType(AccountConfig.class);
+
+    Set<AccountType> adminExpAccountTypeSet =
+        accountTypeRepository
+            .all()
+            .filter("self.technicalTypeSelect not in ('commitment','view')")
+            .fetch()
+            .stream()
+            .collect(Collectors.toSet());
+    accountConfig.setAdminExpAccountTypeSet(adminExpAccountTypeSet);
```

Comment:
no needs to set to accountConfig as its just a context object and value get already set with next statement